buildspec/linuxIntegrationTests.yml

```diff
@@ -9,7 +9,7 @@ env:
 phases:
     install:
         runtime-versions:
-            nodejs: 10
+            nodejs: 12
```

Comment:
yargs requires node 12 now:

    /codebuild/output/src266610087/src/github.com/aws/aws-toolkit-vscode/node_modules/yargs/build/index.cjs:1
    Error: yargs supports a minimum Node.js version of 12. Read our version support policy: https://github.com/yargs/yargs#supported-nodejs-versions